src/Uno.UI/UI/Xaml/Controls/Page/Page.Android.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Android.Views;
+using Android.Widget;
+using Uno.Logging;
+using Uno.Extensions;
+using Uno.UI.DataBinding;
+using Uno.UI.Controls;
+using Windows.UI.Xaml.Data;
+using System;
+using System.Collections.Generic;
+using Uno.Disposables;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using System.Drawing;
+using System.Linq;
+using Uno.UI;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class Page
+	{
+		private BorderLayerRenderer _borderRenderer = new BorderLayerRenderer();
+
+		private void InitializeBorder()
+		{
+			Loaded += (s, e) => UpdateBorder();
+			Unloaded += (s, e) => _borderRenderer.Clear();
+			LayoutUpdated += (s, e) => UpdateBorder();
+		}
+
+		private void UpdateBorder()
+		{
+			UpdateBorder(false);
+		}
+
+		private void UpdateBorder(bool willUpdateMeasures)
+		{
+			if (IsLoaded)
+			{
+				_borderRenderer.UpdateLayers(
+					this,
+					Background,
+					Thickness.Empty,
+					null,
+					CornerRadius.None,
```

Comment:
should we be using `Control.CornerRadius` here? What does UWP do? (probably not commonly used on a `Page` I admit)

Reply:
Corner radius, border thickess and brushes are not supported on Page.